hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/SimpleFeesSuite.java

```diff
@@ -274,10 +272,8 @@ final Stream<DynamicTest> cryptoCreatePlainComparison() {
                             .balance(0L)
                             .payingWith(PAYER)
                             .fee(ONE_HBAR)
-                            .via("create-account-txn"),
-                    // node=1, network=2, service=0.00000022 (22 tinycents)
-                    // sigs = 1 (included), keys = 0
-                    validateChargedUsdWithin("create-account-txn", ucents_to_USD(3), 0.01));
+                            .via("createAccountTxn"),
+                    validateChargedUsdWithin("createAccountTxn", 0.06, 1.0));
```

Comment:
USD price for `cryptoCreate` is `0.05`. Once we fix the simpleFeeSchedules, if we are still charging `0.06` we should take a look